openpype/tools/project_manager/project_manager/widgets.py

```diff
@@ -141,16 +144,25 @@ def __init__(self, parent=None, dbcon=None):
         inputs_widget = QtWidgets.QWidget(self)
         project_name_input = QtWidgets.QLineEdit(inputs_widget)
         project_code_input = QtWidgets.QLineEdit(inputs_widget)
+        project_width_input = QtWidgets.QLineEdit(inputs_widget)
+        project_height_input = QtWidgets.QLineEdit(inputs_widget)
+        project_fps_input = QtWidgets.QLineEdit(inputs_widget)
         library_project_input = QtWidgets.QCheckBox(inputs_widget)
 
         inputs_layout = QtWidgets.QFormLayout(inputs_widget)
         inputs_layout.setContentsMargins(0, 0, 0, 0)
         inputs_layout.addRow("Project name:", project_name_input)
         inputs_layout.addRow("Project code:", project_code_input)
         inputs_layout.addRow("Library project:", library_project_input)
+        inputs_layout.addRow("Width:", project_width_input)
+        inputs_layout.addRow("Height:", project_height_input)
+        inputs_layout.addRow("FPS:", project_fps_input)
 
         project_name_label = QtWidgets.QLabel(self)
         project_code_label = QtWidgets.QLabel(self)
+        project_width_label = QtWidgets.QLabel(self)
+        project_height_label = QtWidgets.QLabel(self)
```

Comment:
I don't think having strings for resolution width and height is a good idea - these should always be integers?

Also, I feel like _if_ these fields are added to the create project page that it should at the very least default to your studio's default settings for a project too.

Reply:
100% agree. There should be `QSpinBox` and should have values from studio default anatomy values.

Reply:
Great idea about filling the default values, will take a look how to do that. About the integers inputs, do you think I need to use spinbox for that?

Reply:
Well `QSpinBox` will take care about the value being valid integer, also gives user hint that the values should be integer.